raster/src/main/scala/geotrellis/raster/RasterExtent.scala

```diff
@@ -241,10 +241,10 @@ object RasterExtent {
     RasterExtent(extent, cw, ch, cols, rows)
   }
 
-  def apply(extent: Extent, cellwidth: Double, cellheight: Double): RasterExtent = {
-    val cols = (extent.width / cellwidth).toInt
-    val rows = (extent.height / cellheight).toInt
-    RasterExtent(extent, cellwidth, cellheight, cols, rows)
+  def apply(extent: Extent, cellSize: CellSize): RasterExtent = {
```

Comment:
Yes, I ran into a surprising problem: When giving two longs (like from TileLayout) it will choose the Double, Double overload rather than Int, Int. It eventually broke because CellSize was 255,255 for LatLng extent, but it was not at all obvious why at first.